<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>275</width>
				<height>500</height>
			</rect>
		</property>
		<property name="sizePolicy">
			<sizepolicy hsizetype="Fixed" vsizetype="Fixed">
				<horstretch>0</horstretch>
				<verstretch>0</verstretch>
			</sizepolicy>
		</property>
		<property name="minimumSize">
			<size>
				<width>275</width>
				<height>500</height>
			</size>
		</property>
		<property name="maximumSize">
			<size>
				<width>275</width>
				<height>500</height>
			</size>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<property name="styleSheet">
			<string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
				background-color: rgb(244, 221, 192);
				color:rgb(0, 0, 0);
			</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<widget class="QLabel" name="module_selection_TITLE">
						<property name="maximumSize">
							<size>
								<width>16777215</width>
								<height>25</height>
							</size>
						</property>
						<property name="text">
							<string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span
								style=&quot; font-size:12pt;&quot;&gt;Select Module to View Channels&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
							</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module1">
						<property name="enabled">
							<bool>true</bool>
						</property>
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 1</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module2">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 2</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module3">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="maximumSize">
							<size>
								<width>200</width>
								<height>16777215</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 3</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module4">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 4</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module5">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 5</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module6">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 6</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module7">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 7</string>
						</property>
					</widget>
				</item>
				<item alignment="Qt::AlignHCenter">
					<widget class="QPushButton" name="module_selection_Module8">
						<property name="minimumSize">
							<size>
								<width>150</width>
								<height>0</height>
							</size>
						</property>
						<property name="styleSheet">
							<string notr="true">background-color:rgb(204, 51, 0);
								color: rgb(255, 255, 255);
								font: 12pt &quot;MS Shell Dlg 2&quot;;
							</string>
						</property>
						<property name="text">
							<string>DAQ Module 8</string>
						</property>
					</widget>
				</item>
			</layout>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
